profile-widget/divecartesianaxis.cpp

```diff
@@ -409,7 +409,7 @@ QColor TimeAxis::colorForValue(double value)
 
 QString TimeAxis::textForValue(double value)
 {
-	int nr = value / 60;
+	int nr = lrint(value) / 60;
```

Comment:
so now you are first rounding the value to a long, and then are doing the fraction / modulo. That seems correct.